linkerd/identity/src/lib.rs

```diff
@@ -405,11 +429,37 @@ impl rustls::ResolvesServerCert for CertResolver {
     }
 }
 
+// === impl LocalId ===
+
+impl From<Name> for LocalId {
+    fn from(n: Name) -> Self {
+        Self(n)
+    }
+}
+
+impl Into<Name> for LocalId {
+    fn into(self) -> Name {
+        self.0
+    }
+}
+
+impl AsRef<Name> for LocalId {
+    fn as_ref(&self) -> &Name {
+        &self.0
+    }
+}
+
+impl fmt::Display for LocalId {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        self.0.fmt(f)
```

Comment:
TIOLI: should this indicate that this is a local identity? i guess the `Debug` impl does that so maybe this shouldn't?

Reply:
Correct. This is really for encoding the name into labels, etc. We want to encode the underlying name string and not something more descriptive.